### PR TITLE
Timestamp system variable

### PR DESCRIPTION
**Pull request by briehl**
the token system var removal part is covered in a separate PR, but this builds on it.

### CODE REVIEW

**Comment by briehl** [comment]
Once this gets tested and merged, I'll propagate it down to the py3-update branch.

File: src/biokbase/narrative/app_util.py [hunk 1]
```diff
@@ -77,6 +73,12 @@ def system_variable(var):
         except:
             return None
         # TODO: make this better with more exception handling.
+    elif var == 'timestamp_epoch_ms':
+        # get epoch time in milliseconds
+        return int(time.time()*1000)
```
Comment (MrCreosote) [inline src/biokbase/narrative/app_util.py]: Can't you, you know, actually get the time in ms rather than multiplying sec by 1K?

Comment (briehl) [reply src/biokbase/narrative/app_util.py]: Nope. Either seconds or (in python 3.7) ns. The seconds function has floating point accuracy out to ns, though, depending on the OS. On the ones I've tried (and on what the Narrative runs), it's fine.

https://docs.python.org/3/library/time.html#time.time

Comment (MrCreosote) [reply src/biokbase/narrative/app_util.py]: Right, I forgot it returned floating point. Sorry for the noise.

**Pull request merged**